### PR TITLE
Added function to collect stats about all classes for a classifier

### LINKED ISSUE #1247: Aggregation of classifier fields of all time
### Short Description
Parent issue: #1241.

Introduce a function that allows fetching of stats about all the classes for a given classifier.

### PR DESCRIPTION
**Pull request by singhpratyush**
### Short description

Fixes #1247.

I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by singhpratyush** [comment]
Please do not merge yet.

**Comment by singhpratyush** [comment]
@Achint08 @djmgit @hemantjadon @kavithaenair @SarishiNoHara @SKrPl @vibhcool: Open for review.

The build fails because of some issue in `RedirectUnshortener`. It is happening at development branch too.

File: src/org/loklak/data/ElasticsearchClient.java [hunk 1]
```diff
@@ -501,19 +504,19 @@ public int deleteByQuery(String indexName, final QueryBuilder q) {
...
         return map;
     }
-    
+
     /**
      * Write a json document into the search index.
      * Writing using a XContentBuilder is the most efficient way to add content to elasticsearch
-     * 
+     *
      * @param jsonMap
```
Comment (SKrPl) [inline src/org/loklak/data/ElasticsearchClient.java]: There is no parameter with name `jsonMap`.

File: src/org/loklak/data/ElasticsearchClient.java [hunk 2]
```diff
@@ -943,4 +946,46 @@ public Query(final String indexName, final QueryBuilder queryBuilder, String ord
         }
         return list;
     }
+
+    public HashMap<String, HashMap<String, Long>> classifierScore(String index, String classifierName, ArrayList<String> classes) {
```
Comment (SKrPl) [inline src/org/loklak/data/ElasticsearchClient.java]: Please add Javadoc comments for the method.

File: src/org/loklak/data/ElasticsearchClient.java [hunk 3]
```diff
@@ -943,4 +946,46 @@ public Query(final String indexName, final QueryBuilder queryBuilder, String ord
...
+            HashMap<String, Long> map = new HashMap<>();
+            map.put("count", docCount);
+            map.put("sum", (long) sum.getValue());
+            map.put("avg", (long) avg.getValue());
+            aggregatedMap.put(key,  map);
+        }
+        return aggregatedMap;
+    }
+
+    public TermsBuilder getClassifierAggregation(String classifierName) {
```
Comment (SKrPl) [inline src/org/loklak/data/ElasticsearchClient.java]: Same, please add Javadoc comments here.

File: src/org/loklak/data/ElasticsearchClient.java (hunk 2 above)
Comment (SKrPl) [inline src/org/loklak/data/ElasticsearchClient.java]: Can the declaration be broken down into multiple lines like one parameter per line, as it exceeds 100 chars per line?

**Pull request merged**